controllers/parameters.go

```diff
@@ -30,6 +30,9 @@ func buildParameters(kubeClient client.Client, namespace string, parametersFrom
 				return nil, nil, err
 			}
 			for k, v := range fps {
+				if k == "shared" {
```

Comment:
please add a comment explaining this code

Reply:
ok

Reply:
the comment is not clear and misleading, sm supports shared property..

Reply:
Made it clearer